### DISCUSSION TITLE
Provide a warning when deleting an IOC that is referenced in an event timeline

### DISCUSSION THREAD

**Discussion by cudeso** [dfir-iris/iris-web#41] [Ideas]
When an IOC is referenced in an event timeline there should be a warning when you try to delete the IOC.

**Comment by whikernel** [comment]
Hello ! 

For now the IOCs aren't actually referenced in the events. The timeline is enhanced locally in JS with the list of IOCs. 
So issuing a warning would imply to search server-side each IOC in title and content of every event, which is not the most effective. 

We also tried to reduce user interactions as much as possible during IR.   
But I understand that a warning might prevent accidental deletions. @cudeso Is that the reason why you created this issue ?

Let's see if more people want this feature, in which case we'll implement. We'll let the issue open for now !

**Comment by cudeso** [comment]
Hi! Thank you for the feedback. Indeed, it was to prevent accidental deletions.
